litmus-portal/frontend/src/components/Header/ProfileDropdownItems.tsx

```diff
@@ -56,11 +57,25 @@ function ProfileInfoDropdownItems(props: ProfileInfoDropdownItemProps) {
   }
 
   const [loggedOut, doLogout] = useState(false);
-
+  const { userData } = useSelector((state: RootState) => state);
   const logOut = () => {
     doLogout(true);
     user.userLogout();
-    history.push('/');
+
+    fetch(`http://3.9.117.22:30375/logout`, {
```

Comment:
This endpoint would be a taken from config right! `${config.auth.url}/logout`

Reply:
Yeah, missed it.